scripts/shellcheck.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -e
+
+EXCLUDE_RULES="SC1090,SC1091,SC2001,SC2064,SC2181,SC1117,SC2164"
+
```

Comment:
Was curious about the exclusions. shellcheck is passing without these exclusions:

https://www.shellcheck.net/wiki/SC1090
https://www.shellcheck.net/wiki/SC1117
https://www.shellcheck.net/wiki/SC2181

And we might not want to exclude (but it would require minor fixes)
https://www.shellcheck.net/wiki/SC2064 (use single quotes) - one error in securedrop-keyring
https://www.shellcheck.net/wiki/SC2164 (use `cd ... || exit` in case cd fails)

Reply:
Thanks, I copied this list from sd-server, I'll remove the ones that are already passing and single quotes. I added the cd one because in my review nearly all (or all) of the uses of cd were for expected paths, so I think constantly adding || exit is going to be really annoying.